src/methods/arraybuffer.js

```diff
@@ -133,12 +136,12 @@ export function DetachArrayBuffer(realm: Realm, arrayBuffer: ObjectValue): NullV
 // ECMA262 24.2.1.1
 export function GetViewValue(
   realm: Realm,
-  view: Value,
+  _view: Value,
   requestIndex: Value,
   isLittleEndian: Value,
   type: ElementType
 ): NumberValue {
-  view = view.throwIfNotConcrete();
+  let view = _view.throwIfNotConcrete();
```

Comment:
I don't think there is a need to distinguish _view and view.

Reply:
I do this because flow strict doesn't allow mutation of function parameters, and the linting rules don't allow re-declaration.

Reply:
I see. That is a bit sad. Never mind then.